phases/4/README.md

```diff
@@ -51,6 +51,10 @@ As with phases 1, 2, and 5, weekly (cycle-based) project tracking for learners i
 
 [trello]: https://trello.com/b/5y3fu6Xj/phase-4
 
-## Development Guide
+## Getting Started
 
-[Git Workflow](development-guide.md)
+To begin preparing for your work in Phase 4, you should read the following documentation and follow its instructions:
+
+- [learning-os-software README file](https://github.com/jrpool/learning-os-software)
+- [Learning OS software best practices, global requirements, and architecture](https://github.com/jrpool/learning-os-software/blob/master/SUMMARY.md) 
```

Comment:
Can we move this content into markdown files within this repo? we can just put them under 'phases/4/learning-os-software`

Reply:
These link to a large amount of content. I'm hesitant to add this to the curriculum without having read it all. Can we pull out a subset and put it in this repo so I and others can help iterate on in?

Thanks!

Reply:
Apologies for a big mistake: I didn’t notice that the links were internal to my fork, instead of linking to the original repo. Assuming that you were reacting to the impression that this is new documentation, I am going to submit a new PR with corrected links.